libs/ui/src/modals/screens/tokens-selector/tokens-selector.tsx

```diff
@@ -33,27 +34,33 @@ const keyExtractor = ({ tokenAddress, tokenId }: TokenType) => getTokenSlug(toke
 export const TokensSelector: FC = () => {
   const allTokensMarketInfoSelector = useTokensMarketInfoSelector();
   const {
-    params: { token }
-  } = useRoute<RouteProp<ScreensParamList, ScreensEnum.SendTokensSelector>>();
+    params: { token, field, withBalanceTokens }
+  } = useRoute<RouteProp<ScreensParamList, ScreensEnum.TokensSelector>>();
   const { navigate } = useNavigation();
   const { chainId } = useSelectedNetworkSelector();
   const visibleAccountTokens = useVisibleAccountTokensSelector();
-  const accountTokensWithBalance = useMemo(() => getTokensWithBalance(visibleAccountTokens), [visibleAccountTokens]);
+  const accountTokens = useMemo(
```

Comment:
`accountTokensWithBalanceAndGasToken` useMemo and `accountTokens` useMemo could be merged into 1 useMemo

Reply:
on send screen, we show all tokens with balance and gasToken, and gasToken can be without balance, so I think it can not be merged